### DISCUSSION TITLE
The discussions tab doesn't have a button to start a discussion

### DISCUSSION THREAD

**Discussion by majorhayes** [community/community#29] [Discussions] [answered]
I tried to use the discussions feature for a repository per the repo's instructions, but there was no way to create a discussion.
![image](https://user-images.githubusercontent.com/39497236/97069902-c5075c80-1590-11eb-86fd-bb492b991b7a.png)

I assume there's a reason for not being able to create a discussion, but there was no explanation anywhere to help me understand why. It would be very helpful to provide users with obvious feedback when the option to create a new discussion is disabled so we don't end up going crazy wondering why we can't find the button.

**Comment by becca** [comment, marked as answer]
There is a control at the organization level which can limit Discussions so that only those with triage+ permissions can open new discussions. Anyone can still comment on an existing/opened discussion. 

If possible, I would recommend you reach out to the maintainers of this organization to see if they intended to have this control disabled.

**Reply by majorhayes** [reply]
The point i was trying to make is that it wasn't clear to me as a user that it was just disabled. This was my first attempt to even use this feature on GitHub and it was a very frustrating experience because  I spent a bunch of time hunting around for the button and doing google searches to try to figure out how to add a discussion.

I think it would be good UX to indicate somewhere that I simply don't have permissions to create a discussion, perhaps by having the button grayed out with a hover text or something like that.

**Reply by becca** [reply]
Thank you for clarifying @majorhayes and providing this feedback. Will keep you posted on providing a clear solution to this problem. 🙇

**Reply by rseanhall** [reply]
I would also like to see this addressed, because it's also missing when the user isn't signed in. For issues, the New Issue button is always visible but makes you sign in when you click it.

**Reply by becca** [reply]
Thanks for calling that out @rseanhall. We'll look into improving the logged out state for new discussion creation flow as well.

**Reply by cjbarth** [reply]
> There is a control at the organization level which can limit Discussions so that only those with triage+ permissions can open new discussions. Anyone can still comment on an existing/opened discussion. 
> 
> If possible, I would recommend you reach out to the maintainers of this organization to see if they intended to have this control disabled.

Fixed
